MSTest SDK doc: use latest released version (#43714)
dotnet/docs: docs/core/testing/unit-testing-mstest-sdk.md

```

> [!NOTE]

> > `/3.6.3` is given as example as it's the first version of the SDK, but it can be replaced with any newer version.

Reviewer: this note is wrong after the update.

> > `/3.6.3` is given as example as it's the first version of the SDK, but it can be replaced with any newer version.

Author: ```suggestion
> `/3.6.3` is given as example and can be replaced with any newer version.
```
